packages/cli/src/credentials/credentials.service.ts

```diff
@@ -52,11 +55,21 @@ export class CredentialsService {
 
 		// if member, return credentials owned by or shared with member
 
-		const userSharings = await Db.collections.SharedCredentials.find({
+		const whereConditions: FindManyOptions = {
```

Comment:
Something is very odd here.

I expect this should be `FindManyOptions<SharedCredentials> & { where: ObjectLiteral }` and no assertion below, much like the other fix, but for some reason `where` here is not being narrowed.

This is not worth more than a few minutes, so I set it aside for now. Feel free to disregard. Also below.